baseband/vdif/frame.py

```diff
@@ -318,9 +320,17 @@ def data(self):
     def size(self):
         return len(self.frames) * self.frames[0].size
 
+    @property
```

Comment:
Okay, this makes me more amenable to having more instances of sample shape throughout the code, though if I read correctly you only use `frameset.shape`, so we can still just do:
```
@property
def shape(self):
    return (self.shape[0], len(self.frames))  + self.shape[1:]
```